.github/workflows/algolia-index.yml

```diff
@@ -10,6 +10,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: getsentry/action-setup-volta@c52be2ea13cfdc084edb806e81958c13e445941e # v1.2.0
+      - uses: dorny/paths-filter@v3
+        id: filter
+        with:
+          filters: |
+            docs:
+              - 'docs/**'
```

Comment:
Can we have multiple items here?

docs are affected by includes as well

Otherwise we can have a `dev-docs` otherwise default to `docs`